cmd2/utils.py

```diff
@@ -93,18 +97,20 @@ def str_to_bool(val: str) -> bool:
 
 
 class Settable:
-    """Used to configure a cmd2 instance member to be settable via the set command in the CLI"""
+    """Used to configure an attribute to be settable via the set command in the CLI"""
 
     def __init__(
         self,
         name: str,
-        val_type: Callable,
+        val_type: Union[Type[Any], Callable[[Any], Any]],
         description: str,
         *,
-        onchange_cb: Callable[[str, Any, Any], Any] = None,
-        choices: Iterable = None,
-        choices_provider: Optional[Callable] = None,
-        completer: Optional[Callable] = None
+        settable_object: Optional[object] = None,
```

Comment:
What is the use case for this being None?

Reply:
All legacy settables have this set to None.